src/applications/appeals/10182/config/form.js

```diff
@@ -197,7 +197,7 @@ const formConfig = {
       title: 'Board review option',
       pages: {
         boardReviewOption: {
-          title: 'Board review option',
+          title: 'Board review option ',
```

Comment:
When the chapter title and page title are _exactly_ the same, the page title won't render (see screenshot). So adding a trailing space makes them different.

Reply:
Is this a bug that needs fixing?
I think a comment to this effect would be good as that would totally be something I would clean up if I saw it, not knowing why it was there.

Reply:
It's built into the form system, so I think it was intentional - they didn't want the same chapter and title to be read twice (in screen readers). Ideally, our chapter title and page title should be different, but that likely involves CAIA. @eileen-coforma WDYT?

Reply:
@eileen-coforma @Mottie It's worth making the fix as proposed to fix the visual issue. Agree that it's not ideal that we have back to back h3 and h4 with the same text. I don't think this would block a Veteran, but it might confuse them.

Maybe we could eventually change the h3 text to "Board review type" or something similar. That might be tough given forms library constraints.

Reply:
Ok, if that's how we want to fix it then let's slap a comment over it explaining why it's there (and perhaps linking back to this convo)